lib/active_interaction/filters/file_filter.rb

```diff
@@ -4,8 +4,9 @@ module ActiveInteraction
   class Base
     # @!method self.file(*attributes, options = {})
     #   Creates accessors for the attributes and ensures that values passed to
-    #   the attributes respond to the `eof?` method. This is useful when passing
-    #   in Rails params that include a file upload or another generic IO object.
+    #   the attributes respond to the `rewind` method. This is useful when
+    #   passing in Rails params that include a file upload or another generic
+    #   IO object.
```

Comment:
I think we should leave it in. Without it, we basically say that the file input accepts any IO-like object. That's not very clear to me. Plus we couldn't move away from `rewind` until we support only Rails >=4.0.2, which might be a while.